src/com/sun/jna/NativeLibrary.java

```diff
@@ -109,7 +109,12 @@ private NativeLibrary(String libraryName, String libraryPath, long handle, Map<S
             synchronized(functions) {
                 Function f = new Function(this, "GetLastError", Function.ALT_CONVENTION, encoding) {
                         @Override
-                        Object invoke(Object[] args, Class<?> returnType, boolean b) {
+                        Object invoke(Object[] args, Class<?> returnType, boolean b, int fixedArgs) {
```

Comment:
I recently changed the lowest-level "invoke" signature when handling interface-based mappings in order to accommodate varargs correctly, as part of a varargs fix for `arm`.  This is transparent to any existing usage, but I neglected to update the signature on this `GetLastError` short-circuit, with the result being that _some_ (all?) calls directly to `GetLastError` would no longer be short-circuited.